lib/spectcl.js

```diff
@@ -327,7 +330,7 @@ Spectcl.prototype.expect = function(expArr, cb){
 
           if(expectationCb){
               debug('[expect] [%d] calling handler for %s', self.child.pid, matchedExpectation)
-              cbRetVal = expectationCb(matchedExpectation, finalCb)
+              cbRetVal = expectationCb(matchedExpectation, finalCb, expectMatch[matchedExpectation])
```

Comment:
Agreed, callbacks are always last. We should pass the match second and bump to `v0.6.0` in this change.